### PR TITLE
fix validate_coinbase

### PR DESCRIPTION
**Pull request by Jeanmichel7**
- [x] issue #86 
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/raito/blob/main/CONTRIBUTING.md)
- [x] code change includes tests

### CODE REVIEW

**Comment by Jeanmichel7** [comment]
It was intended to add the outputs to the utreexoset, but this no longer seems the best place.

File: src/validation.cairo [hunk 1]
```diff
@@ -290,19 +290,11 @@ fn validate_coinbase(block: @Block, total_fees: u64, block_height: u32) -> Resul
...
-        .txs {
-            for output in *txs
-                .outputs {
-                    total_output_amount = total_output_amount + *output.value;
-                    //TODO add outputs to UtreexoSet
-
-                };
-        };
+    for output in *tx.outputs {
+        total_output_amount = total_output_amount + *output.value;
```
Comment (TAdev0) [inline src/validation.cairo:296]: total_output_amount += *output.value would be more concise

Comment (Jeanmichel7) [reply src/validation.cairo:296]: That's true I had seen it too, and forgot to change it

**Review by TAdev0 — APPROVED** [review]
Lgtm!

**Review by m-kus — APPROVED** [review]

**Pull request merged**